e2e-tests/tests/gr-tls-cert-manager/02-assert.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 300
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: gr-tls-cert-manager-ca-cert
+spec:
+  commonName: gr-tls-cert-manager-ca
+  duration: 8760h0m0s
+  isCA: true
+  issuerRef:
+    kind: Issuer
+    name: gr-tls-cert-manager-pso-ca-issuer
+  renewBefore: 730h0m0s
+  secretName: gr-tls-cert-manager-ca-cert
+status:
+  conditions:
+    - message: Certificate is up to date and has not expired
+      observedGeneration: 1
+      reason: Ready
+      status: 'True'
+      type: Ready
+  revision: 1
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: gr-tls-cert-manager-ssl
+spec:
+  issuerRef:
+    kind: Issuer
```

Comment:
Is this enough to check that it was published by cert manager? IIRC there were some labels or annotations which mentioned cert manager specifically.

Reply:
there is nothing like that at least by the operators request
```
apiVersion: cert-manager.io/v1
kind: Certificate
metadata:
  name: gr-tls-cert-manager-ca-cert
  namespace: kuttl-test-new-whale
  uid: 4d8a5c45-b734-4f60-8dd1-d26efdb6c7b0
  resourceVersion: '200109'
  generation: 1
  creationTimestamp: '2023-01-23T15:15:11Z'
  ownerReferences:
    - apiVersion: ps.percona.com/v1alpha1
      kind: PerconaServerMySQL
      name: gr-tls-cert-manager
      uid: 04e058d1-9088-4d75-b67d-ce6f923c7a24
      controller: true
      blockOwnerDeletion: true
spec:
  commonName: gr-tls-cert-manager-ca
  duration: 8760h0m0s
  isCA: true
  issuerRef:
    kind: Issuer
    name: gr-tls-cert-manager-pso-ca-issuer
  renewBefore: 730h0m0s
  secretName: gr-tls-cert-manager-ca-cert
status:
  conditions:
    - lastTransitionTime: '2023-01-23T15:15:12Z'
      message: Certificate is up to date and has not expired
      observedGeneration: 1
      reason: Ready
      status: 'True'
      type: Ready
  notAfter: '2024-01-23T15:15:12Z'
  notBefore: '2023-01-23T15:15:12Z'
  renewalTime: '2023-12-24T05:15:12Z'
  revision: 1

```

Reply:
Ok